src/System.Numerics.Vectors/src/System/Numerics/Quaternion.cs

```diff
@@ -155,9 +155,9 @@ public static Quaternion Inverse(Quaternion value)
         }
 
         /// <summary>
-        /// Creates a Quaternion from a vector and an angle to rotate about the vector.
+        /// Creates a Quaternion from a unit vector axis and an angle to rotate about the vector.
         /// </summary>
-        /// <param name="axis">The vector to rotate around.</param>
+        /// <param name="axis">The unit vector to rotate around. This must be a vector of length 1.</param>
```

Comment:
Either of those are ok, but I think we should briefly explain what happens if you don't do that, e.g. "This should be a vector of length 1 or else the result will be non-sensical." or something better worded.